protogen/src/main/java/com/netflix/conductor/protogen/Element.java

```diff
@@ -0,0 +1,111 @@
+package com.netflix.conductor.protogen;
+
+import com.netflix.conductor.common.annotations.ProtoEnum;
+import com.netflix.conductor.common.annotations.ProtoMessage;
+import com.netflix.conductor.protogen.types.AbstractType;
+import com.netflix.conductor.protogen.types.MessageType;
+import com.squareup.javapoet.MethodSpec;
+import com.squareup.javapoet.TypeSpec;
+
+import java.util.*;
+
+public abstract class Element {
```

Comment:
javadoc, what is this for? why is it in protogen package if it is manually created?

Reply:
I got it now, protogen package is not for generated types, but rather for generator that generates proto files from existing  types, great idea!